apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -465,6 +474,25 @@ const aichatSlice = createSlice({
       state.fieldVisibilities =
         levelAichatSettings?.visibilities || DEFAULT_VISIBILITIES;
     },
+    resetToDefaultAiCustomizations: (
+      state,
+      action: PayloadAction<{
+        levelAichatSettings?: LevelAichatSettings;
+      }>
+    ) => {
+      const {levelAichatSettings} = action.payload;
+
+      const defaultAiCustomizations: AiCustomizations =
+        levelAichatSettings?.initialCustomizations || EMPTY_AI_CUSTOMIZATIONS;
+
+      state.savedAiCustomizations = defaultAiCustomizations;
+      state.currentAiCustomizations = defaultAiCustomizations;
+      state.fieldVisibilities =
+        levelAichatSettings?.visibilities || DEFAULT_VISIBILITIES;
+      state.chatMessages = [];
```

Comment:
Could we dispatch a separate action(s) to do the 1) clearing of chat history and 2) adding a new message? These feel distinct from the stated "resetToDefaultAiCustomizations". Or at least put the `clearChatMessages` logic (ie, empty chat mesages list, reset session id) somewhere shared in redux so we don't lose track that this is a place where we also reset a session?

Thinking `resetProject` could be:

```
  const resetProject = useCallback(() => {
    dispatch(
      resetToDefaultAiCustomizations({
        levelAichatSettings,
      })
    );
    dispatch(clearChatMessages());
    dispatch(addChatMessage(the notification));
  }, [dispatch, levelAichatSettings]);
```

Reply:
Was debating about this with myself so good with your suggestion!